BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -241,10 +241,11 @@ public final class BlueprintView: UIView {
     private func updateViewHierarchyIfNeeded() {
         guard needsViewHierarchyUpdate || bounds != lastViewHierarchyUpdateBounds else { return }
 
-        assert(
+        precondition(
```

Comment:
This seems like it should always fire, even in prod.

Reply:
err, do you think so? does this cause undefined behavior or will things still work? if things still work, not crashing seems better

Reply:
Generally we want our assertions to align between prod and debug (this is why POS includes `NSAssert` in prod); and yes it could cause undefined behavior because we do not actually ever test this case.